testgrid/config/config.yaml

```diff
@@ -874,6 +874,9 @@ test_groups:
 # azure
 - name: kubernetes-azure
   gcs_prefix: kube_azure_log/kubernetes.azure.nightly
+# Canonical Distribution of Kubernetes (contact: @chuckbutler on github)
+- name: canonical-kubernetes-e2e-gce
+  gcs_prefix: canonical-kubernetes-tests/logs/gce/e2e-node
```

Comment:
This layout is a little unusual-- can you put the logs in canonical-kubernetes-tests/logs/gce-e2e-node instead? If not, you should change the entry in buckets.yaml to be logs/gce as well, since it expects the layout to be like:

$PATH/$JOB/$BUILD/started.json

The 24-hour test history collector won't recursively search for job directories, so it won't handle your current results being stored in $PATH/gce/$JOB/$BUILD